composer.json

```diff
@@ -24,7 +29,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^5.2",
-        "squizlabs/php_codesniffer": "^2.3"
+        "squizlabs/php_codesniffer": "^2.3",
+        "aws/aws-sdk-php": "^3.36"
```

Comment:
SES is a very old service and as a consequence I think we should not be as tight in the dependency. I think a slightly lower version should be enough.

Reply:
Here I need 3.36.25 because of templated email management

Reply:
Good point! Let's keep it this way then.